rhel6/xccdf/system/accounts/restrictions/root_logins.xml

```diff
@@ -162,6 +162,38 @@ cause the system to become inaccessible.
 <ref nist="AC-2" disa="178" />
 </Rule>
 
+<Rule id="no_password_auth_for_systemaccounts" severity="medium">
```

Comment:
Why is this a medium severity? Shouldn't it be low, given there are no logins permitted to system accounts in the first place?

Reply:
Edit: Nevermind. I see this is just to fall in line with DISA